agones/src/lib.rs

```diff
@@ -75,10 +78,23 @@ async fn setup_namespace(client: kube::Client) -> String {
     let lp = ListParams::default().labels("owner=quilkin-test");
     let nss = namespaces.list(&lp).await.unwrap();
     let dp = DeleteParams::default();
+
+    let delay = env::var_os(DELETE_DELAY_SECONDS)
+        .map(|value| chrono::Duration::seconds(value.into_string().unwrap().parse().unwrap()));
```

Comment:
Nit: You don't need to use `var_os` if you're converting to string anyway, and you can flatten the unwraps.

```suggestion
    let delay = env::var(DELETE_DELAY_SECONDS)
        .as_deref()
        .ok()
        .and_then(|s| i64::parse(s).ok())
        .map(chrono::Duration::seconds);
```

Reply:
I swear I never saw `env::var()` existing before. Totally missed that. Thank you!